packages/console/src/components/PlanDescription/index.tsx

```diff
@@ -10,20 +9,26 @@ const registeredPlanDescriptionPhrasesMap: Record<
 > = {
   [ReservedPlanId.Free]: 'free_plan_description',
   [ReservedPlanId.Pro]: 'pro_plan_description',
-  [ReservedPlanId.Enterprise]: 'enterprise_description',
+};
+
+const getRegisteredPlanDescriptionPhrase = (
+  skuId: string,
+  isEnterprisePlan = false
+): TFuncKey<'translation', 'admin_console.subscription'> | undefined => {
+  if (isEnterprisePlan) {
+    return 'enterprise_description';
+  }
+
+  return registeredPlanDescriptionPhrasesMap[skuId];
 };
 
 type Props = {
-  /** Temporarily mark as optional. */
-  readonly skuId?: string;
-  /** @deprecated */
-  readonly planId: string;
+  readonly skuId: string;
+  readonly isEnterprisePlan?: boolean;
 };
 
-function PlanDescription({ skuId, planId }: Props) {
-  const description =
-    conditional(skuId && registeredPlanDescriptionPhrasesMap[skuId]) ??
-    registeredPlanDescriptionPhrasesMap[planId];
+function PlanDescription({ skuId, isEnterprisePlan = false }: Props) {
```

Comment:
Want to double check the usage scenarios of the component `<PlanDescription />`, if this component is used NOT only when displaying the "current plan description," the passed parameters might be problematic: `isEnterprisePlan` indicates whether "the current plan" is an enterprise plan. If the component is used to display "past plan descriptions," this parameter could lead to incorrect display content.

Reply:
This component is used in two places: `SelectTenantPlanModel` and `CurrentPlan`.

The `isEnterprisePla`n parameter doesn’t inherently indicate whether it applies to the current plan. Regardless of whether the provided skuId represents the current plan or a past plan, it should still be able to indicate if it’s an enterprise plan with `isEnterprisePlan`.

Reply:
If it's only used in the two scenarios you mentioned, this change seems fine to me.